### ISSUE TITLE
How can I finetune on previous model trained before

### ISSUE THREAD

**Issue by rainylt** [matterport/Mask_RCNN#2144]
I have trained a mask rcnn model before and it seems good. But now I want to add some new data with the same category to finetune it. The new data include 20 images which are 1280*960 size with 300 objects, and the images used before are smaller with about 30 object.

I have modified the config.py as #1884. But after one epoch trained on Head, the model perform worse, and after 20 epoches, the model can only detect few objects on images. 
Is it normal that the model perform worse when I add new images?

Here is the code I trained on new images.
```
model.train(dataset_train, dataset_val,
            learning_rate=config.LEARNING_RATE,#lr=0.01
            epochs=20,
            layers='heads')
```

In addition, the new images contain many objects some of which are quiet small, and the loss with new images is more than 6 in the first epoch.
